### PR TITLE
fixing ecr public region

### PR DESCRIPTION
**Pull request by serventera**

### CODE REVIEW

**Review by sakti — APPROVED** [review]
LGTM

File: .github/workflows/docker-image.yml [hunk 1]
```diff
@@ -127,7 +127,7 @@ jobs:
       with:
         aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
         aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-        aws-region: ap-southeast-1
+        aws-region: us-east-1
```
Comment (mochadwi) [inline .github/workflows/docker-image.yml:130]: yaaah ECR unavailable di ASEAN region kah mas? @serventera @sakti

Comment (serventera) [reply .github/workflows/docker-image.yml:130]: iya itu ecr public global soale jadi loginnya harus ke us-east region. ref: https://docs.aws.amazon.com/AmazonECR/latest/public/getting-started-cli.html#:~:text=The%20Amazon%20ECR%20Public%20registry%20requires%20authentication%20in%20the%20us%2Deast%2D1%20Region

Comment (mochadwi) [reply .github/workflows/docker-image.yml:130]: noted, lgtm mas thanks!

**Pull request merged**